Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -100,7 +104,11 @@ public bool IsActiveBuildTargetValid()
             { Configurations.ForceTextSerialization, new ConfigGetter(() => { return IsForceTextSerialization(); }) },
             { Configurations.VisibleMetaFiles, new ConfigGetter(() => { return IsVisibleMetaFiles(); }) },
             { Configurations.VirtualRealitySupported, new ConfigGetter(() => { return XRSettingsUtilities.LegacyXREnabled; }) },
-            { Configurations.SinglePassInstancing, new ConfigGetter(() => { return MixedRealityOptimizeUtils.IsSinglePassInstanced(); }) },
+#if UNITY_ANDROID
+            { Configurations.SinglePass,  new ConfigGetter(() => { return MixedRealityOptimizeUtils.IsSinglePass(); }) },
+#else
+            { Configurations.SinglePassInstancing,  new ConfigGetter(() => { return MixedRealityOptimizeUtils.IsSinglePassInstanced(); }) },
+#endif
```

Comment:
per comment above, change to 

{ Configurations.OptimalRenderingPath, new CONfigGetter(() =>
{
if Android
do blah
else
do regular
endif
}};